etc/alignak.backend-import.cfg

```diff
@@ -0,0 +1,210 @@
+# Alignak configuration file for importing data in the Alignak backend
+# --------------------------------------------------------------------
+
+# -------------------------------------------------------------------------
+# Monitored objects configuration part
+# -------------------------------------------------------------------------
+# Configuration files with common objects like commands, timeperiods,
+# or templates that are used by the host/service/contacts
+; When loading data only from the alignak backend, comment this
+; block because data do not need to be read from files
+cfg_dir=arbiter/objects/realms
+cfg_dir=arbiter/objects/commands
+cfg_dir=arbiter/objects/timeperiods
+cfg_dir=arbiter/objects/escalations
+cfg_dir=arbiter/objects/dependencies
+
+# Templates and packs for hosts, services and contacts
+; When loading data only from the alignak backend, comment this
+; block because data do not need to be read from files
+cfg_dir=arbiter/templates
+cfg_dir=arbiter/packs
+
+# Notification ways
+; When loading data only from the alignak backend, comment this
+; block because data do not need to be read from files
+cfg_dir=arbiter/objects/notificationways
+
+# Groups
+; When loading data only from the alignak backend, comment this
+; block because data do not need to be read from files
+cfg_dir=arbiter/objects/servicegroups
+cfg_dir=arbiter/objects/hostgroups
+cfg_dir=arbiter/objects/contactgroups
+
+# Real hosts, services and contacts
+; When loading data only from the alignak backend, comment this
+; block because data do not need to be read from files
+cfg_dir=arbiter/objects/hosts
+cfg_dir=arbiter/objects/services
+cfg_dir=arbiter/objects/contacts
+
+# Alignak daemons and modules are loaded
+; commented because they may disturb configuration loading for importation
+; and there is no need to load daemons and modules configuration
+; cfg_dir=arbiter/daemons
+; cfg_dir=arbiter/modules
+; but we must declare our own arbiter with no modules in its configuration
+define arbiter {
```

Comment:
No. It must not ... this file is perfect for importing into the backend. I tested it hundedth times ... anyway, I removed it from Alignak repo.